scipy/spatial/_geometric_slerp.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import division, print_function, absolute_import
+
+__all__ = ['geometric_slerp']
+
+import math
+import numpy as np
+from . import _slerp
+from scipy.spatial.distance import euclidean
+
+def geometric_slerp(start,
+                    end,
+                    t,
+                    tol=1e-7):
+    """
+    Spherical linear interpolation along unit-radius
```

Comment:
Nitpick --- the recommended formatting is one line followed by a more thorough explanation.